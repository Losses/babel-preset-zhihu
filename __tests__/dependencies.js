import {transform} from '@babel/core'

test('transform dependencies', () => {
  expect(
    transform(`const fn = () => {}`, {
      envName: 'development',
      configFile: false,
      presets: ['./dependencies'],
    }).code,
  ).toMatchSnapshot()
})
