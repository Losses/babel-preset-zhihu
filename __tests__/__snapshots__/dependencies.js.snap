// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`transform dependencies 1`] = `"var fn = function fn() {};"`;
